main.py

```diff
@@ -1,7 +1,47 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
+
+
+def get_firstname_lastname_from_full_name(fullname: str) -> dict:
+    data = fullname.split()
+    return {"first_name": data[0], "last_name": data[1]}
 
 
 def main() -> QuerySet:
-    pass
+    genres_to_create = ["Western", "Action", "Dramma"]
+    for curr_item in genres_to_create:
+        Genre.objects.create(
+            name=curr_item
+        )
+
+    actors_to_create = [
+        get_firstname_lastname_from_full_name("George Klooney"),
+        get_firstname_lastname_from_full_name("Kianu Reaves"),
+        get_firstname_lastname_from_full_name("Scarlett Keegan"),
+        get_firstname_lastname_from_full_name("Will Smith"),
+        get_firstname_lastname_from_full_name("Jaden Smith"),
+        get_firstname_lastname_from_full_name("Scarlett Johansson"),
+    ]
+    for actor_data in actors_to_create:
+        Actor.objects.create(
+            **actor_data
+        )
```

Comment:
And use unpacking to create actors that making code more readable and avoiding multiple function calls
```suggestion
    for first_name, last_name in actors_to_create:
        Actor.objects.create(
            first_name=first_name,
            last_name=last_name
        )
```